index.html

```diff
@@ -54,5 +55,156 @@ <h2 class="start-btn">Start collaboration</h2>
+          <li>CSS</li>
+          <li>JavaScript</li>
+        </ul>
+        <div class="more-info">
+          <button type="button" class="button-styles">See this project <i class="fas fa-arrow-right"></i></button>
+        
+        </div>
+      </div>
+      <div class="project-img">
+        <img src="images/second.png" class="image-2" alt="" />
+      </div>
+      <div class="project-details">
+        <h2 class="project-info">Project name goes here</h2>
+        <ul class="info">
+          <li>HTML</li>
+          <li>CSS</li>
+          <li>JavaScript</li>
+        </ul>
+      </div>
+      <div class="more-info">
+        <button type="button" class="button-styles">See this project <i class="fas fa-arrow-right"></i></button>
+    
+      </div>
+      <div class="project-img">
+        <img src="images/third.png" class="image-3" alt="" />
+      </div>
+      <div class="project-details">
+        <h2 class="project-info">Project name goes here</h2>
+        <ul class="info">
+          <li>HTML</li>
+          <li>CSS</li>
+          <li class="info">JavaScript</li>
+        </ul>
+      </div>
+      <div class="more-info">
+        <button type="button" class="button-styles">See this project <i class="fas fa-arrow-right"></i></button>
+        
+      </div>
+
+      <div class="project-img">
+        <img src="images/fouth.png" class="image-4" alt="" />
+      </div>
+
+      <div class="project-details">
+        <h2 class="project-info">Project name goes here</h2>
+        <ul class="info">
+          <li>HTML</li>
+          <li>CSS</li>
+          <li class="info">JavaScript</li>
+        </ul>
+      </div>
+      <div class="more-info">
+        <button type="button" class="button-styles">See this project <i class="fas fa-arrow-right"></i></button>
+      
+      </div>
+      <div class="project-img">
+        <img src="images/fifth.png" class="image-5" alt="" />
+      </div>
+      <div class="project-details">
+        <h2 class="project-info">Project name goes here</h2>
+        <ul class="info">
+          <li>HTML</li>
+          <li>CSS</li>
+          <li class="info">JavaScript</li>
+        </ul>
+      </div>
+      <div class="more-info">
+        <button type="button" class="button-styles">See this project <i class="fas fa-arrow-right"></i></button>
+        
+      </div>
+      <div class="project-img">
+        <img src="images/six.png" class="image-6" alt="" />
+      </div>
+      <div class="project-details">
+        <h2 class="project-info">Project name goes here</h2>
+        <ul class="info">
+          <li>HTML</li>
+          <li>CSS</li>
+          <li>JavaScript</li>
+        </ul>
+      </div>
+      <div class="more-info">
+        <button type="button" class="button-styles">See this project <i class="fas fa-arrow-right"></i></button>
+        
+      </div>
+    </section>
+    <section class="main-info">
+      <div class="main-info-div">
+        <h1 class="header-3">About</h1>
```

Comment:
- [ ] Nice working on the About us section kindly add some `line-space` in between the paragraph text so it aligns with the Figma Template.
- [ ] Kindly add padding to the list in the `.more-info` cards and make the text to be bolder, so it aligns with the Figma Template.

## Figma Template
![abt-1](https://user-images.githubusercontent.com/48371156/166115869-4da56eb8-d6c9-4826-84df-bfe33e8d0768.png)

## Your Design
![abt-2](https://user-images.githubusercontent.com/48371156/166115874-7831b004-808c-4659-ab13-7bf705bcc2bd.png)